spec.bs

```diff
@@ -9177,20 +9229,54 @@ requests into smaller number of fetches. It's a [=struct=] with the following [=
   :: A [=list=] of [=trusted scoring signals requests=] that hasn't yet been organized to aid
     batching.
   : <dfn>request map</dfn>
-  :: A [=map=] from a tuple of [=script fetcher=], a [=URL=] representating the trusted signals
+  :: A [=map=] from a tuple of [=script fetcher=], a [=URL=] representing the trusted signals
     base [=URL=], {{unsigned short}} or null for experiment ID, [=origin=] or null for coordinator,
-    and [=origin=], representing the top frame's [=origin=], to a [=list=] of [=trusted scoring
+    and [=origin=], representing the top frame's [=origin=], and a [=boolean=] representing whether
+    to send creative scanning info, to a [=list=] of [=trusted scoring
     signals requests=]. This organizes fetches that can possibly be merged together.
   : <dfn>url length limit</dfn>
   :: A {{long}} denoting a user-configured limit which should not be exceeded due to combining of
     fetches. 0 denotes no limit, and is the initial value.
 </dl>
 
+A <dfn>creative info</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="creative info">
+  : <dfn>ad descriptor</dfn>
+  :: An [=ad descriptor=].
+  : <dfn>creative scanning metadata</dfn>
+  :: A [=string=], initially empty.

```

Comment:
just to confirm, it means empty string, not null, right?

Reply:
Yes. It's also a string, not a string-or-null.